<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableSummaries</class>
 <widget class="QWidget" name="TableSummaries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TableSummaries</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,0,0">
     <item>
      <widget class="QLabel" name="label_Icon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/icons/icons/glyphicons_072_bookmark.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Title">
       <property name="text">
        <string>发货记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_FilterMask">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>图表类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_ChartType">
       <property name="frame">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>饼图</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/icons/glyphicons_042_pie_chart.png</normaloff>:/icons/icons/glyphicons_042_pie_chart.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>趋势图</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/icons/glyphicons_040_stats.png</normaloff>:/icons/icons/glyphicons_040_stats.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>柱状图</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/icons/glyphicons_041_charts.png</normaloff>:/icons/icons/glyphicons_041_charts.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Filter">
       <property name="text">
        <string>筛选 (&amp;F)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_027_search.png</normaloff>:/icons/icons/glyphicons_027_search.png</iconset>
       </property>
       <property name="shortcut">
        <string>F3</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QTableView" name="tableView_Records">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>200</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="widget_Chart" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Print">
       <property name="text">
        <string>打印 (&amp;P)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_015_print.png</normaloff>:/icons/icons/glyphicons_015_print.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ExportImage">
       <property name="text">
        <string>导出图片 (&amp;M)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/icons/glyphicons_138_picture.png</normaloff>:/icons/icons/glyphicons_138_picture.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+M</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
  <include location="C:/Python27/Lib/site-packages/PyQt4/resource.qrc"/>
 </resources>
 <connections/>
</ui>
